frontend/routes/viewer/components/views/views.component.tsx

```diff
@@ -0,0 +1,300 @@
+/**
```

Comment:
@dleszcz Please move view item code to separate component ;)

Reply:
done